resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/backend/minikube/Minikube.scala

```diff
@@ -34,6 +34,7 @@ private[spark] object Minikube extends Logging {
   private val MINIKUBE_VM_PREFIX = "minikubeVM: "
   private val MINIKUBE_PREFIX = "minikube: "
   private val MINIKUBE_PATH = ".minikube"
+  private val MINIKUBE_APISERVER_CERT_PATH = ".minikube"
```

Comment:
Shall we remove this? This is unused, isn't it?

Reply:
Ah, right. I'll remove it.